### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by l-jonas**
> Do you want to remove the release scripts then? And make a new release?

(https://github.com/syncthing/syncthing-lite/pull/68#issuecomment-436243519)

@Nutomic Yes. I removed the scripts and documented the release process here. But there is one missing thing: The release to Google Play. It looks like there is some process according to <https://github.com/syncthing/syncthing-lite/issues/66>.

### CODE REVIEW

**Comment by Nutomic** [comment]
Right, I'm currently trying to get the Google Play build working on the build server.

**Comment by Nutomic** [comment]
I'm not done yet with the build server, but once that's complete, it should build and publish new releases automatically to Google Play (unless I misunderstand the config). It will also need some changes to the gradle files, and a new release with those changes. I will make a PR with that as soon as it's working.

**Comment by Nutomic** [comment]
Okay the build server is working for release builds, I merged the changes directly in a294d6f06cf73a303a7927b8f57b1f9833f8734a because they were quite minor. It should publish to Google Play automatically once a release is tagged.

By the way we could also use this to update Google Play screenshots, description and translations directly from files in git (or Transifex), like we already do for Syncthing Android.

**Comment by l-jonas** [comment]
@Nutomic For managing the store assets, someone who manages needs to import the current store assets using ``./gradlew bootstrap`` (https://github.com/Triple-T/gradle-play-publisher#quickstart). But I don't know how the store entry text files can be connected with Transifex.

I didn't find anything about changelogs in the documentation of the gradle-play-publisher.

Do you mean by "disabled" that it's in the mode where only users who already downloaded it can see and download it again? This would explain <https://github.com/syncthing/syncthing-lite/issues/66>. Or is it that it's required to manually confirm each release in the Google Play Console?

**Comment by Nutomic** [comment]
Right, I will take care of this later. Changelogs can also be updated, see [here](https://github.com/Triple-T/gradle-play-publisher#uploading-release-notes). It needs a seperate file, so I'm not sure yet how that might work with the Github changelog.

The app is currently unpublished, so it won't show up in the store at all. I will publish it again once we have a new release to publish there (I don't think it makes sense to release now with missing translations and known bugs).

File: Releasing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+- do tests
+- update translations using ``tx pull -a -af`` (as extra merge request or branch for the case it does not build correctly)
+- update the version name and version code of the app
+- create a tag/ release in GitHub with an changelog; The tag name should be the version number
+
+## notes
+
+- F-Droid will pick up the new release automatically
+- The upload to google play happnes using the CI when a release was tagged
```
Comment (Nutomic) [inline Releasing.md]: Also automatically, and F-Droid also picks it up based on the tag.

**Comment by l-jonas** [comment]
Is en-US set as default language for Google Play? (For documenting which file/ which language should be used for the changelog)

**Pull request merged**